code/datums/emergency_calls/clf.dm

```diff
@@ -1,7 +1,8 @@
 //Colonial Liberation Front
 /datum/emergency_call/clf
 	name = "CLF Cell"
-	base_probability = 0
+	base_probability = 20
+	alignement_factor = -1
 
```

Comment:
Actually thi should be 0 since clf hate both sides

Reply:
Applied.